app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -506,7 +506,7 @@ BrowserWorld::Draw() {
   if (!m.glInitialized) {
     m.glInitialized = m.context->InitializeGL();
     if (!m.glInitialized) {
-      VRB_LOG("FAILED to initialize GL");
+      VRB_ERROR("FAILED to initialize GL");
```

Comment:
Due to the Android lifecycle, this isn't really an error. Probably okay to leave as a VRB_LOG.